test/helpers/loki/receiver.go

```diff
@@ -179,8 +180,7 @@ func (r *Receiver) Query(logQL string, orgID string, limit int) ([]StreamValues,
 // QueryUntil repeats the query until at least n lines are received.
 func (r *Receiver) QueryUntil(logQL string, orgID string, n int) (values []StreamValues, err error) {
```

Comment:
Might want to add context.Context to the Query() method and pass it on to the http request like above. For direct calls to Query(), call Query(context.TODO()...)

Reply:
I find 13. This change is primary meant to update dependencies and remove dead code.  I'd like to limit the scope as much as possible